src/mbgl/util/color.cpp

```diff
@@ -17,4 +18,12 @@ optional<Color> Color::parse(const std::string& s) {
     }};
 }
 
+std::string Color::stringify() const {
+    return "rgba(" +
```

Comment:
Does it matter that a `Color` created via an `hsla()` string would get converted back to JSON as an `rgba()` string?